### ISSUE TITLE
Cannot find validation dataset

### ISSUE THREAD

**Issue by yhhhli** [lyh983012/ES-imagenet-master#2]
Hello,

Thanks for the open-sourced code. However, I had trouble finding the validation set. I directly download the frame set in your cloud server. However, I direct uncompress the file and I didn't find the validation dataset. Also, your `dataset_generator/vallabel.txt` is empty. How can I find the validation index file and the dataset?

Thanks.

**Comment by lyh983012** [comment]
I'm so sorry for this bug, and I have uploaded the correct validation label file _dataset_generator/vallabel.txt_ .  BTW in which link did you download the dataset? I need to check the file

**Comment by lyh983012** [comment]
_Validation Set.zip_ package is uploaded in the first link [https://cloud.tsinghua.edu.cn/d/94873ab4ec2a4eb497b3/](url).

**Comment by yhhhli** [comment]
Thanks, will try it. last question, how long will it take to a model? Thanks again!

**Comment by lyh983012** [comment]
I've used 8*2080Ti to train a 8-time-step LIF-SNN, and it takes about 4 days to reach the top Acc. (~30 epochs). If you try the ANN based models, it will be much faster i guess. You can try the SNN models under _./pretrained_model_ using _./botebook/test_acc.ipynb_ (CUDA is recommended)
